docs/embedding.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 7
+---
+
+# Embedding Apex into Apps
+
+## Libraries
+
+Apex natively supports TypeScript and Go.
+
+* [@apexlang/core](https://github.com/apexlang/apex-js) - TypeScript Apex parser used by the CLI and VS Code extension
+* [@apexlang/codegen](https://github.com/apexlang/codegen) - TypeScript code generation modules loaded by the Apex CLI (e.g. `apex generate`)
+* [github.com/apexlang/apex-go](https://github.com/apexlang/apex-go) - Go Apex parser for embedding in Go platforms
+
+## WebAssembly module
+
+To reach a broader set of languages, the Apex Go parser is also released as a WebAssembly module and loadable via a Wasm runtime such as [Wasmtime](https://wasmtime.dev). The module parsers and validates an Apex specification and returns a JSON representation that is easier to interpret in languages with JSON support.
+
+### Download
+
+The latest **apex-parser.wasm** can be downloaded from the [Apex Go releases page](https://github.com/apexlang/apex-go/releases). The JSON representation is [documented here using Apex](https://github.com/apexlang/apex-go/blob/main/model.apexlang).
+
+### Parsing and validating
```

Comment:
This entire section is counter to the simplicity that Apexlang is attempting to advocate.  There must be an easier way to use the wasm library.

Reply:
I added waPC support that I can mention now